Sources/tart/Commands/Push.swift

```diff
@@ -115,6 +124,19 @@ struct Push: AsyncParsableCommand {
     return RemoteName(host: registry.host!, namespace: registry.namespace,
                       reference: Reference(digest: digest))
   }
+  // Helper method to convert labels array to dictionary
+  func parseLabels() -> [String: String] {
+    var result = [String: String]()
+    for label in labels {
+      let parts = label.components(separatedBy: "=")
+      guard parts.count == 2 else { continue }
```

Comment:
I think it makes sense to support empty values too.

Otherwise we're loosing some information provided by the user.

Reply:
Improved in https://github.com/cirruslabs/tart/pull/1052/commits/b74929ebafa72899a6464b805fe8f4672c9e04fa.